sdks/python/apache_beam/examples/complete/game/hourly_team_score.py

```diff
@@ -216,21 +99,22 @@ def expand(self, pcoll):
         # need to weed out events that fall after the time period we want to
         # analyze.
         | 'FilterStartTime' >> beam.Filter(
-            lambda element: element['timestamp'] > start_min_filter)
+            lambda elem: elem['timestamp'] > self.start_timestamp)
         | 'FilterEndTime' >> beam.Filter(
-            lambda element: element['timestamp'] < end_min_filter)
+            lambda elem: elem['timestamp'] < self.stop_timestamp)
+
         # Add an element timestamp based on the event log, and apply fixed
         # windowing.
-        # Convert element['timestamp'] into seconds as expected by
-        # TimestampedValue.
         | 'AddEventTimestamps' >> beam.Map(
-            lambda element: TimestampedValue(
-                element, element['timestamp'] / 1000.0))
-        # Convert window_duration into seconds as expected by FixedWindows.
-        | 'FixedWindowsTeam' >> beam.WindowInto(FixedWindows(
-            size=self.window_duration * 60))
+            lambda elem: beam.window.TimestampedValue(elem, elem['timestamp']))
+        | 'FixedWindowsTeam' >> beam.WindowInto(
+            beam.window.FixedWindows(self.window_duration_in_seconds))
+
         # Extract and sum teamname/score pairs from the event data.
-        | 'ExtractTeamScore' >> ExtractAndSumScore('team'))
+        | 'ExtractTeamScores' >> beam.Map(
```

Comment:
Are we planning to change java example as well? I would prefer the two examples to be as similar as possible. Is there a reason for us to not use `ExtractAndSumScore`?